### PR TITLE
feat(myaccounts): add states for denied docs

### PR DESCRIPTION
**Pull request by MatoPlus**
### Notion ticket link

[No criminal check denied flow for myaccount page](https://www.notion.so/uwblueprintexecs/No-criminal-check-denied-flow-for-myaccount-page-6598e06c51f74c8cbcc059418651bc92)

### Implementation description

Parent - Income Proof for Discount:

poi_denied

Volunteer - Criminal Check Approval Status

bgc_denied

- Consider state for null cases as pending and false as declined
- Fix bug introduced in #201 where grade is expected to be a required number

### Steps to test

1. Upload doc as volunteer, as admin, deny the doc, as parent, we should see the case pictured above
2. vice versa for parent

### Checklist

-   [X] My PR name is descriptive and in imperative tense
-   [X] I have run the linter
-   [X] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -54,8 +54,9 @@
         "dateSubmitted": "Date submitted: {{date}}",
         "poi_approved": "Your income statement has been approved. Please be on the lookout for an email from SDC with the discount for future registrations.",
         "poi_pending": "Your income statement is under review.",
+        "poi_declined": "You have been declined for low income discounts. Please contact SDC if you think this is a mistake or submit a new income statement.",
         "bgc_approved": "Your criminal record check has been approved.",
-        "bgc_pending": "Your criminal record check is under review.",
```
Comment (feng-eric) [inline public/locales/en/common.json:58]: Is this not being used anymore?

Comment (MatoPlus) [reply public/locales/en/common.json:58]: we have three states now, so this is also being used.

**Review by bekahma — APPROVED** [review]
looks good! thanks for making the changes :)

File: public/locales/en/common.json (hunk 1 above)
Comment (feng-eric) [reply public/locales/en/common.json:58]: Perhaps I'm misunderstanding but it seems with the new changes, it's only `bgc_approved` and `bgc_declined`, with `bgc_pending` being removed?

Comment (MatoPlus) [reply public/locales/en/common.json:58]: holup yea that shouldn't be removed lol. mb

**Review by feng-eric — APPROVED** [review]

**Pull request merged**